### PR TITLE
Text changes by Maike

### PR DESCRIPTION
**Pull request by tglomb**

### CODE REVIEW

File: scrnaseq.Rmd [hunk 1]
```diff
@@ -335,9 +339,25 @@ if (length(error_messages)) stop(paste(c("", paste("*", error_messages)), collap
 ```
 
 # Read data
+The workflow can be run for pre-processed 10x data, as well as for pre-processed SmartSeq-2 or other data that are represented by a simple table with transcript counts per gene and cell. 
 
 ## Read and print mapping statistics
-We begin by printing mapping statistics that have been produced prior to this workflow. 
+Prior to this workflow, raw sequencing reads were mapped to the reference transcriptome. The resulting mapping statistics are printed below for a first estimation of sample quality.
```
Comment (ktrns) [inline scrnaseq.Rmd:345]: I will re-read all textual changes once we test the PR and generate an HTML. I find that easier to read.

File: scrnaseq.Rmd [hunk 2]
```diff
@@ -771,8 +824,21 @@ sc_cells_to_exclude_summary = purrr::map_dfr(sc_cells_to_exclude, function(s) {
   return(as.data.frame(purrr::map(s, length))) 
   })
 rownames(sc_cells_to_exclude_summary) = names(sc_cells_to_exclude)
-knitr::kable(sc_cells_to_exclude_summary, align="l", caption="Number of excluded cells") %>% 
-  kableExtra::kable_styling(bootstrap_options=c("striped", "hover")) 
+
```
Comment (ktrns) [inline scrnaseq.Rmd:827]: There is no updated HTML yet, so we'll wait to see how this looks like in the report.

Comment (ktrns) [reply scrnaseq.Rmd:827]: @kosankem Can you tell us what your motivation for this change was? 

The issue here is that the filter summary is not generic any more, i.e. if the filter parameter is not nFeature or percent_mt, we will not see it as far as I can tell from the code.

File: scrnaseq.Rmd [hunk 3]
```diff
@@ -1267,8 +1433,13 @@ if (length(levels(sc$orig.ident)) > 1) {
 # Choose the number of PCs to use for clustering
 sc = Seurat::FindNeighbors(sc, dims=1:param$pc_n, verbose=FALSE)
 
+```
+
+```{r clustering_resolution_test, warning=FALSE, fig.height=5, fig.width=12}
 # Seurat vignette suggests resolution parameter between 0.4-1.2 for datasets of about 3k cells
-sc = Seurat::FindClusters(sc, resolution=param$cluster_resolution, algorithm=4, verbose=FALSE, method="igraph")
+for (i in param$cluster_resolution_test) {
```
Comment (ktrns) [inline scrnaseq.Rmd:1440]: We would like to adapt the concept, but change the code a bit. We can do that when we merge the PR. Ok?

**Review by ktrns — COMMENTED** [review]
Dear Maike, 

Thanks again for your update and help! We suggest that after you replied to our rounds of comments, we will test and update the code a bit, and then do the merge. 

Best wishes
Katrin

File: scrnaseq.Rmd (hunk 3 above)
Comment (kosankem) [reply scrnaseq.Rmd:1440]: Ok, you are welcome to change it.

File: scrnaseq.Rmd (hunk 2 above)
Comment (kosankem) [reply scrnaseq.Rmd:827]: The idea was to extant the table to include a percentage of how many cells and features remain after the filtering step. I apologize if I made it to stringend at any step. Please do not hesitate to change anything accordingly or to let me know your suggestions in regads to what I should try to change.

Comment (andpet0101) [reply scrnaseq.Rmd:827]: It should be pretty easy but can you send us an example HTML so that we know how it should look like?

Comment (kosankem) [reply scrnaseq.Rmd:827]: Here is an example for the table:  
Example_table

**Pull request merged**